service/src/main/java/bio/terra/pipelines/common/utils/PipelineVariableTypesEnum.java

```diff
@@ -43,24 +43,24 @@ public <T> T cast(String fieldName, Object value, TypeReference<T> typeReference
     }
 
     @Override
-    public String validate(String fieldName, Object value) {
+    public String validate(String fieldName, String fileSuffix, Object value) {
       if (cast(fieldName, value, new TypeReference<Integer>() {}) == null) {
         return "%s must be an integer".formatted(fieldName);
       }
       return null;
     }
   },
-  VCF {
+  FILE {
     @Override
     public <T> T cast(String fieldName, Object value, TypeReference<T> typeReference) {
       return (T) STRING.cast(fieldName, value, new TypeReference<>() {});
     }
 
     @Override
-    public String validate(String fieldName, Object value) {
+    public String validate(String fieldName, String fileSuffix, Object value) {
```

Comment:
do we think we will have a use case where we allow for multiple suffixes or we dont care about the suffix at all? if so how will this method handle those use cases

Reply:
great questions. for now I'm ok supporting exactly one suffix (and I should add a test for that in the database test) and adding support for none or multiple if/when we need it. do you have strong feelings otherwise?

Reply:
hmm  yea i would say for the multiple use case not worth thinking about.  I could go either way on the suffix-less use case.  I dont think any of the possible imputation pipeline inputs we get from users would be a suffix-less use case.  If this is true then yea i would say coding for one speicfic output for every FILE input makes sense

Reply:
cool. technically currently if we have a file input and don't want to check for a suffix, we can have the fileSuffix be an empty string and the validate check will pass. (I just added a test for this)